projects/plugins/jetpack/modules/theme-tools/featured-content.php

```diff
@@ -390,6 +390,10 @@ public static function hide_featured_term( $terms, $taxonomies, $args ) {
 				return $terms;
 			}
 
+			if ( empty( $taxonomies ) ) {
```

Comment:
Feel free to ignore, unnecessary use of `empty()` is one of my pet peeves.
```suggestion
			// WordPress defines the parameter as `array`, but it passes null if `get_terms( $args )` was called
			// without a 'taxonomy' in $args.
			if ( ! is_array( $taxonomies ) ) {
```